config/initializers/sprockets.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+Sprockets.register_compressor 'application/javascript', :terser, Terser::Compressor
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

Reply:
Fixed